data/daemon.conf

```diff
@@ -25,3 +25,6 @@ VerboseDomains=
 
 # Update the message of the day (MOTD) on device and metadata changes
 UpdateMotd=true
+
+# For some plugins, enumerate only devices supported by metadata
+OnlySupported=false
```

Comment:
This makes it sound like we only show supported devices, when I really think it needs to convey "allow plugins to filter by supported" as only very few use the FWUPD_DEVICE_FLAG_ONLY_SUPPORTED filter. Maybe `AllowOnlySupported` or even something out there like `EnumerateAllDevices=true` which seems more semantically valid. Other ideas welcome.

Reply:
I think of those my preference is `EnumerateAllDevices`.  I can adjust if you think that's good.

Reply:
EnumerateAllDevices wfm.